fiddler/manifest.json

```diff
@@ -0,0 +1,46 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "ee617671-508e-4bb3-ba25-8815b11a16aa",
+  "app_id": "fiddler",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows",
+    "Category::Alerting",
+    "Category::Monitoring",
+    "Category::Metrics"
+  ],
+  "display_on_public_website": false,
```

Comment:
`display_on_public_website` will need to be set to `true` to appear on the documentation site.

Reply:
Done.